webapp/src/styles.css

```
@@ -1,353 +1,577 @@
+    var(--color-surface-light-elevated) 84%,
+    black 16%
+  );
+  --line: color-mix(in srgb, #1d2420 13%, transparent);
+  --line-strong: color-mix(in srgb, #1d2420 23%, transparent);
+  --text: #161a17;
+  --muted: #626d66;
+  --accent: var(--accent-700);
+  --accent-soft: color-mix(in srgb, var(--accent-300) 27%, transparent);
+  --danger: #a53d3d;
+  --shadow: 0 1.5rem 4rem color-mix(in srgb, #59645d 19%, transparent);
+  color-scheme: light;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)